Simplenote/src/main/res/values/strings.xml

```diff
@@ -36,8 +36,8 @@
     <string name="website">Visit simplenote.com</string>
     <string name="more_info">More Information</string>
     <string name="note_pinned">Note pinned.</string>
-    <string name="note_selected">%d note selected</string>
-    <string name="notes_selected">%d notes selected</string>
+    <string name="note_selected" tools:ignore="PluralsCandidate">%d note selected</string>
+    <string name="notes_selected" tools:ignore="PluralsCandidate">%d notes selected</string>
```

Comment:
Both the `note_selected` and `notes_selected` resources are referenced in the [`plurals`](https://github.com/Automattic/simplenote-android/blob/8ae53102bedeace1f843017d5bf84b98001094db/Simplenote/src/main/res/values/plurals.xml#L4-L5) file so we don't want to add the `tools:ignore="PluralsCandidate"` attribute to them.  The changes to the `strings.xml` file in this pull request can be reverted.

Reply:
Hey Tyler, thanks for the feedback!
I thought it'd be nice to get rid of those warnings in `strings.xml` file since those strings are already referenced in the `plurals.xml` file as you mention.
I've reverted the changes to `strings.xml` ;)